edx_proctoring/templates/timed_exam/footer.html

```diff
@@ -2,7 +2,7 @@
 <div class="footer-sequence">
   <h4> {% trans "Can I request additional time to complete my exam?" %} </h4>
     <p>{% blocktrans %}
-      If you have disabilities or are taking the exam in difficult conditions,
+      If you have disabilities,
```

Comment:
@efischer19 Honestly, I'm not sure how translations are handled for the strings that are marked up in the edx-proctoring repo. I will see if I can dig up some history on how this was designed to work. @nedbat My assumption is that the strings that are marked up in this repo would not get extracted during the normal edx-platform i18n extraction process. Can you verify that assumption is correct?